lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -683,6 +684,12 @@ sub upgrade {
     $request->{dbh}->{AutoCommit} = 0;
     my $locale = $request->{_locale};
 
+    for my $preset (LedgerSMB::Upgrade_Tests->get_presets) {
+        my $sth = $request->{dbh}->prepare($preset);
+        $sth->execute()
+            or die 'Failed to execute pre-migration preset ';
```

Comment:
What is a pre-migration preset?

Reply:
I simplified some tests by pre-setting utility tables and routines in pg_temp before doing upgrade tests.